src/PKSim.Core/Model/ExpressionProfile.cs

```diff
@@ -34,15 +33,12 @@ public ExpressionProfile() : base(PKSimBuildingBlockType.ExpressionProfile)
       public virtual string Category
       {
          get => _category;
-         set
-         {
-            _category = value;
-            RefreshName();
-         }
+         set => _category = value;
       }
 
       public override string Name
       {
+         get => ExpressionProfileName(MoleculeName, Species, Category);
```

Comment:
All right, I have removed this also... I am not sure if this is going to be a problem with serialization but I don't think so. That way, we never have to worry about sync